sdk/src/main/java/com/oursky/authgear/EmailClient.kt

```diff
@@ -0,0 +1,8 @@
+package com.oursky.authgear
+
+data class EmailClient(val packageName: String) {
+    companion object {
+        @JvmStatic val gmail = EmailClient("com.google.android.gm")
```

Comment:
Constants should be named in UPPER_SNAKE_CASE. See https://kotlinlang.org/docs/coding-conventions.html#property-names